test/functional/questions_controller_test.rb

```diff
@@ -22,12 +22,39 @@ def setup
     assert_response :success
   end
 
-  test 'show note by id' do
+  test 'show question by id' do
     note = nodes(:question)
     get :show, id: note.id
     assert_response :success
   end
 
+  test 'question comment markdown and autolinking works' do
+    node = nodes(:question)
+    assert node.comments.length > 0
+    comment = node.comments.last
+    comment.comment = 'Test **markdown** and http://links.com'
+
+    get :show, id: node.id
+
+    assert_select 'strong', 'markdown'
```

Comment:
@jywarren don't know how to pull this PR to test locally so i cant test. but instead of `"strong"` can you please try `"b"`?